src/compute/servers.rs

```diff
@@ -95,6 +95,8 @@ pub struct NewServer {
     name: String,
     nics: Vec<ServerNIC>,
     block_devices: Vec<BlockDevice>,
+    user_data: Option<String>,
+    config_drive: Option<bool>,
```

Comment:
Note for myself: we should probably move this to creating an `inner` object right away.